### DISCUSSION TITLE
Execute a notification block from javascript code

### DISCUSSION THREAD

**Discussion by jlopez994** [AutomaApp/automa#1355] [Q&A]
Hi,

Is it possible to trigger a notification from javascript block? I'm using this javascript code to run a socket in the client and I'd like to send notifications to the user dinamically. 

The only way I figure out to do this is by triggering another workflow and passing variables, but I'd like to know if there is a simplest way, because this one has some issues when I share the extension.

Thanks in advance.

**Comment by flashvnn** [comment]
I think you can use Javasript block then add notification script to the page, check this link

https://www.javascripttutorial.net/web-apis/javascript-notification/

[Workflow Notification.automa.json.zip](https://github.com/AutomaApp/automa/files/12747741/Workflow.Notification.automa.json.zip)
